lessons/module-2/closures.md

```diff
@@ -148,15 +157,127 @@ Our most thorough definition of a closure is now **when an inner function has ac
 
 In the previous example, you'll notice we could still technically change those grades and snoop on them if we wanted to. This is why we say JavaScript doesn't have a true fashion for creating private variables. We can kind of imply that you shouldn't be fussing with something by hiding it in a function and not exposing that variable declaration outside of it - but we can still gain access to that value. So closures aren't really going to help if you have truly sensitive data that nobody should be able to see.
```

Comment:
The "we can kind of imply...." sentence was really hard for me to understand.  I had to read it several times.   Would this phrasing be easier for students to understand?
"As developers, we can hide a variable declaration inside a function in order to imply that no one should be fussing with that variable. In doing that, we will still be able to gain access to its value."

Reply:
Good suggestion!